src/browser/api/application.js

```diff
@@ -707,6 +693,17 @@ function run(identity, mainWindowOpts, userAppConfigArgs) {
     coreState.setAppRunningState(uuid, true);
 
     ofEvents.emit(route.application('started', uuid), { topic: 'application', type: 'started', uuid });
+
+    const { preloadScripts } = mainWindowOpts;
+    const loadUrl = () => app.mainWindow.loadURL(app._options.url);
```

Comment:
Not sure it matters, but this moves the app started event before the loading of the url, was that intentional?

Reply:
So, so far, it's hasn't affected it in any negative way I could notice while doing automated and manual testing. I do agree though it's probably better to put that event after `loadURL` just like it was before, just to keep things safe.